src/main/webapp/dashboard.html

```diff
@@ -54,10 +58,12 @@ <h1>Admin Dashboard</h1>
     <!--Contains the statistics charts-->
     <div class="right">
       <h1>Site Statistics</h1>
-      <div id="bar-chart" style="width: 900px; height: 500px"></div>
+      <div id="line-chart" style="width: 900px; height: 500px"></div>
+      <br>
+      <br>
       <br>
-      <a href="" id = "redirect" style="text-decoration: none;">go to queue</a>
-      <h2 id = "classCode"> </h2>
+      <a href="" id = "redirect" style="text-decoration: none;">Go to Queue</a>
```

Comment:
Link to redirect is wrong- it should contain the classCode, but the redirect right now has the taEmail